snippets/oadp-image-stream-tag-trigger.adoc

```diff
@@ -0,0 +1,13 @@
+
+//This snippet appears in the following assemblies:
+//
+//backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc
+//OADP 1.4.1 Release Notes
+
+:_mod-docs-content-type: SNIPPET
+
+During a File System Backup (FSB) restore operation, a `Deployment` resource referencing an `ImageStream` is not restored properly. Due to this, pod restoration `postHook` is not executed and the volume content is not restored during. 
+
+This happens because, during the restore operation, an OpenShift controller updates the `spec.template.spec.containers[0].image` field in the `Deployment` resource with the `ImageStreamTag` hash. The update triggers a rollout of a new pod affecting the FSB volume restoration and Pod restoration `postHook` execution. For more information, see "Triggering updates on image stream changes".
```

Comment:
```suggestion
This happens because, during the restore operation, an OpenShift controller updates the `spec.template.spec.containers[0].image` field in the `Deployment` resource with the `ImageStreamTag` hash. The update triggers the rollout of a new pod, affecting the FSB volume restoration and pod restoration `postHook` execution. For more information, see "Triggering updates on image stream changes".
```

Is `Triggering updates on image stream changes` meant to be a link?

Reply:
I have added the link as an xref in the 'Additional  resources'  section of the 'Restoring' assembly. This is per OCP guidelines. Also. we are including the snippet in RN too and the xref will probably fail to resolve.